.github/actions/publish/action.yml

```diff
@@ -28,6 +28,9 @@ runs:
       uses: docker/setup-buildx-action@v3
       with:
         platforms: linux/amd64,linux/arm64/v8,linux/arm/v7,linux/386
```

Comment:
Since we weren't specifying this before, our release binaries would be using whatever version of Go is on the Github Runner